lte/gateway/python/magma/subscriberdb/client.py

```diff
@@ -70,6 +86,7 @@ def __init__(
         subscriber_page_size: int,
         sync_interval: int,
         grpc_client_manager: GRPCClientManager,
+
```

Comment:
there should not be separation between last param and enclosing parentheses on a function

Reply:
resolved as suggested